### PR TITLE
Adding new 50/60fps, No-interlacing, WS patches and fixes

### PR DESCRIPTION
**Pull request by Gabominated**
Adding new patches and  i will expose some of most relevant 

Resident Evil 4 PAL-M 
very tested Widescreen improvement for the poor in-game widescreen

4:3 aspec
Re4 wsoff
in-game widescreen 16:9
Re4 wson
in-game widescreen 16:9 + patch
Re4 wsonpatch

World War Zero - Ironstorm PAL-M
older patch to fix zoom by Arapapa need render, i founf another better zoom that no need for render and forced widescreen cause graphical glitches in pause menu so is better leave only in-game option widescreen to turn on manually

Patch by Arapapa
World War Zero - Ironstorm_SLES-51924_20240204221108
New Patch
World War Zero - Ironstorm_SLES-51924_20240204221127

Resident Evil - CODE Veronica X NTSC-U
older patch by nemesis2000 is not good scaled

4:3 Aspect
Resident Evil - CODE Veronica WSoff
patch by nemesis2000
Resident Evil - CODE Veronica WSolder
new patch ported from PAL-M4 version by ELHecht
Resident Evil - CODE Veronica WSnew

Psi-Ops - The Mindgate Conspiracy NTSC-U
a lot of games interlaced was fixed with new builds but a few still suffering with distortion and shake at videos, logos and others screen, even games like Genji still suffering at gameplay (i'm woking on it) but this title works fine with this patch that use a progressive mode

without patch
Psi-Ops - The Mindgate Conspiracy_SLUS-20688_20240125140538
Psi-Ops - The Mindgate Conspiracy_SLUS-20688_20240126163831
with patch
Psi-Ops - The Mindgate Conspiracy_SLUS-20688_20240125140618
Psi-Ops - The Mindgate Conspiracy_SLUS-20688_20240126164015

adding a few fps patch by asasega tested by myself some with fixes and other supplanted with new codes with better performance

### CODE REVIEW

File: patches/SLUS-20184_24036809.pnach [hunk 1]
```diff
@@ -2,67 +2,8 @@ gametitle=Resident Evil: Code: Veronica X (SLUS-20184)
...
-patch=1,EE,002B9A58,word,460C6B02
-patch=1,EE,002B9A5c,word,3C010050
-patch=1,EE,002B9A60,word,E42C8140
-patch=1,EE,002B9A64,word,E42D8138
-patch=1,EE,002B9A68,word,03E00008
-patch=1,EE,002B9A6c,word,E42E8130
-
-
+author=Gabominated
+description=Widescreen hack ported from PAL-M4 ver by ELHecht
```
Comment (pgert) [inline patches/SLUS-20184_24036809.pnach:6]: Typo: "ELHecht" -> "ElHecht"

File: patches/SLUS-20673_E3ADDC73.pnach [hunk 2]
```diff
@@ -8,4 +8,7 @@ comment=Widescreen Hack by Arapapa
 patch=1,EE,00248178,word,3c013ec0 // 3c013f00 hor fov
 patch=1,EE,001f3c30,word,3c013f40 // 3c013f80 renderfix
 
-
+[60 FPS]
+author=asasega
+description=Unlocked at 60 FPS. Might need enable EE Overclock to be stable.
+patch=1,EE,2024BEF0,extended,14400039 //10400039
```
Comment (pgert) [inline patches/SLUS-20673_E3ADDC73.pnach:14]: The hack by asasega is "patch=1,EE,2024BEF0,word,10000039":
https://forums.pcsx2.net/Thread-60-fps-codes?pid=552039#pid552039

If your hack is better, then I suggest this:
author=asasega & Gabominated

Avoids confusion.

File: patches/SLUS-21006_95CC86EF.pnach [hunk 3]
```diff
@@ -0,0 +1,6 @@
+gametitle=Ghost in the Shell - Stand Alone Complex [SLUS_21006] [2F123FD8]
```
Comment (pgert) [inline patches/SLUS-21006_95CC86EF.pnach:1]: Wrong CRC: "2F123FD8" -> "95CC86EF"

File: patches/SLUS-21037_2BDA8ADB.pnach [hunk 4]
```diff
@@ -1,15 +1,11 @@
...
-//patch=1,EE,001d87b8,word,3c013fe3 //3c013faa
-//patch=1,EE,001d87bc,word,34218e39 //3421aaab
-//patch=1,EE,003327f0,word,3c013fe3 //3c013faa
-//patch=1,EE,003327f4,word,34218e39 //3421aaab
-
-//Causes a crash on Level 2.
-
-
+gametitle=Project - Snowblind SLUS-21037 2BDA8ADB secundary crc 7849F069
+//Corresponds to the main crc to activate the secondary crc 75A5436A called SP.IMG
```
Comment (pgert) [inline patches/SLUS-21037_2BDA8ADB.pnach:2]: ... secundary crc 7849F069 called BO3GM.ELF

Yes?

You have it mixed up with this (I guess):

gametitle=Tom Clancy's Ghost Recon: Advanced Warfighter * NTSC-U * SLUS-21422 * 75A5436A
// CRC shifts from 433B0342 to 75A5436A
// 75A5436A ELF file is called "SP.IMG"

Also, the shift accours when selecting single player mode, not on boot - yes, so...
// CRC shifts from 2BDA8ADB to 7849F069 (BO3GM.ELF) when selecting single player mode.

File: patches/SLUS-20184_24036809.pnach (hunk 1 above)
Comment (Gabominated) [reply patches/SLUS-20184_24036809.pnach]: > Typo: "ELHecht" -> "ElHecht"

Done

File: patches/SLUS-20673_E3ADDC73.pnach (hunk 2 above)
Comment (Gabominated) [reply patches/SLUS-20673_E3ADDC73.pnach:14]: Done

File: patches/SLUS-21006_95CC86EF.pnach (hunk 3 above)
Comment (Gabominated) [reply patches/SLUS-21006_95CC86EF.pnach]: Done

File: patches/SLUS-21037_2BDA8ADB.pnach (hunk 4 above)
Comment (Gabominated) [reply patches/SLUS-21037_2BDA8ADB.pnach]: Done

**Comment by refractionpcsx2** [comment]
The 60 FPS patches which force 480p may be better renamed to `480p 60FPS` just for clarity to what the patch does.

**Comment by pgert** [comment]
> The 60 FPS patches which force 480p may be better renamed to `480p 60FPS` just for clarity to what the patch does.

Suggestion: "[480p & 60 FPS]".

**Comment by Mrlinkwii** [comment]
whats the status ?

**Comment by Gabominated** [comment]
> whats the status ?

hey, i was checking for this pull for a month and I assumed that everything was ok and never check anymore, big surprise that my pull still here lol, anyway i made requested changes

**Pull request merged**